patches/node/chore_handle_default_configuration_not_being_set_in_the_electron_env.patch

```diff
@@ -0,0 +1,19 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@slack-corp.com>
+Date: Tue, 23 Jul 2019 11:36:48 -0700
+Subject: chore: handle default_configuration not being set in the electron env
```

Comment:
should we rather set this up in the env?

Reply:
_mmaayybbeeee_ but our config.gypi doesn't currently support dynamic changes, happy to raise a TODO though to update our GN files to generate the config.gypi 😆